postgres/functions/refresh_matviews.sql

```diff
@@ -4,11 +4,13 @@ CREATE OR REPLACE FUNCTION public.refresh_app_monitoring_matviews()
 AS $function$
 DECLARE
   matview RECORD;
+  start_time timestamp with time zone;
 BEGIN
+  start_time := clock_timestamp();
```

Comment:
Move this into the loop so that you get the actual time each operation took. As currently implemented, you'll be tracking all durations relative to when the transaction started.

Reply:
I should have caught that in the earlier review. After that you should be good to go.